xsd/netex_framework/netex_responsibility/netex_version_support.xsd

```diff
@@ -174,6 +214,11 @@ Rail transport, Roads and Road transport
 				<xsd:documentation>Whether ENTITY is currently in use. Default is "active".</xsd:documentation>
 			</xsd:annotation>
 		</xsd:attribute>
+		<xsd:attribute name="publication" type="PublicationEnumeration" use="optional" default="public">
```

Comment:
Suggestions for a more precise name: publicationSharingPolicy, publicationVisibility